src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -16,6 +16,7 @@
       <SkipUseReferenceAssembly Condition="'$(TargetsWindows)' == 'true'">true</SkipUseReferenceAssembly>
     </ProjectReference>
     <ProjectReference Include="$(LibrariesProjectRoot)System.Security.AccessControl\src\System.Security.AccessControl.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Principal.Windows\src\System.Security.Principal.Windows.csproj" />
```

Comment:
Yes, these cases are now expected as before, this project came in transitively.

Reply:
I was expecting more, but we just hit one.